shell/app/menus/cmp.tsx

```diff
@@ -15,7 +15,7 @@ import i18n from 'i18n';
 import { goTo, insertWhen } from 'common/utils';
 import { filterMenu, MENU_SCOPE } from './util';
 import React from 'react';
-import { ErdaIcon } from 'common';
+import ErdaIcon from 'common/components/erda-icon';
```

Comment:
Why change this import style?

Reply:
the call chain is too long, unexpected errors can occur